src/routes/api/v1/search-index/+server.ts

```diff
@@ -108,6 +90,23 @@ export const POST = async ({ request, locals: { contentfulClient } }) => {
   }
 };
 
+const getMetaTitleAndDescription = (
+  item?: SearchIndexingMetadataMapItem,
+): { metaTitle?: string | null; metaDescription?: string | null } => {
+  let metaTitle, metaDescription;
+  if (item?.__typename == "PageMetadata") {
+    metaTitle = item?.metaTitle ?? item?.title;
+    metaDescription = item?.metaDescription;
+  } else if (item?.__typename === "News") {
+    metaTitle = item?.metaTitle ?? item?.title;
+    metaDescription = item?.metaDescription ?? item?.subhead;
+  } else if (item?.__typename === "EventEntry") {
+    metaTitle = item?.metaTitle ?? item?.shortTitle;
+    metaDescription = item?.metaDescription ?? item?.eventSummary;
+  }
+  return { metaTitle, ...(metaDescription ? { metaDescription } : {}) };
+};
+
```

Comment:
this is only getting used by the GET endpoint below that @larsmaxwell added. I _did_ add a query filter to that endpoint, so if you wanna quickly view just one of the maps, you can do `?filter=pages` or `?filter=news-and-events`. Incomplete but something worth returning to in the future if needed

Reply:
If you're looking to populate the search index, it would be good to have the GET endpoints loading separate content types as well (one for page content/ one for news/events).

Reply:
Since the service entries are already taken care of, and there was only one event to add, this wasn't really needed for a one-time bulk import. But I think it's nice to have around from a utility standpoint in the future